10-Types-of-contributions.md

```diff
@@ -0,0 +1,228 @@
+# Types of contributions
+In this chapter, we will discuss the types of contributions that open-source projects involve.
+
+Mostly, there are two types of contributions. Which are;
```

Comment:
You can remove "Which are;" and just have a colon at the end of the previous sentence

Mostly, there are two types of contributions: